test/textAngularToolbar.spec.js

```diff
@@ -91,6 +91,17 @@ describe('textAngularToolbar', function(){
 		
 		describe('respects the Class attribute taToolbarActiveButtonClass', function(){
 			it('on an active button', inject(function($rootScope, $compile, textAngularManager){
+				//mock range and rangeObject
+				var range = {
```

Comment:
I stedet for at skrive en kommentar omkring mock så kan du kalde dem 
`rangeMock` og `rangeObject` så bliver kode nemlig selvforklarende ;)